### ISSUE TITLE
HTML in emails is not formatted properly

### ISSUE THREAD

**Issue by hkolvenbach** [eclipse-che/che#8970]
### Description
When sending emails with che multiuser, the HTML content is not formatted properly, `<` is replaced with `&lt` (see attached screenshot and copy of the email source code).

If someone could point me to the location where this needs to be fixed, I could give it a try.

Email source:
```
------=_Part_2_922454530.1519904558025
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

<html>
<head>
&lt;style&gt;@import url(https://fonts.googleapis.com/css?family=3DRoboto);=
&lt;/style&gt;
</head>
<body style=3D"width: 700px; margin: 0 auto; font-size: 16px; font-family: =
Roboto;">
<div style=3D"background-color: #292c2f; color: white; padding: 25px 10px;"=
>
    <span style=3D"font-size: 26px; color: #cccccc;">Eclipse Che</span>
    <div style=3D"height: 50px"></div>
    <span style=3D"font-size: 24px; color: white;">Reset your password</spa=
n>
</div>

    <div style=3D"padding: 10px;">
        <p>Someone just requested to change your Che account&#39;s credenti=
als. If this was you, click on the link below to reset them. If you don&#39=
;t want to reset your credentials, just ignore this message and nothing wil=
l be changed.</p>
        &lt;p&gt;&lt;span style=3D&quot;background-color: #fdb940; padding:=
 5px 20px; font-family: arial narrow; font-weight: bold;&quot;&gt;&lt;a sty=
le=3D&quot;color: white; text-decoration: none;&quot; href=3D&quot;http://e=
<dummy-domain>:5050/auth/realms/che/logi=
n-actions/action-token?key=3DeyJhbGciOiJIUzUxMiIsImtpZCIgOiAiNTEyNzZkNGItNW=
I5Yi00NmJiLWE4ZjUtZmI5MzQ4NTVlMjBjIn0.eyJqdGkiOiIzNzFjYjI5Ny0wYjYzLTRjZTAtO=
WY4OS0xMThhZWUyNzQ5YjUiLCJleHAiOjE1MTk5MDQ4NTgsIm5iZiI6MCwiaWF0IjoxNTE5OTA0=
NTU4LCJpc3MiOiJodHRwOi8vZWMyLTM0LTI0MC0yMzYtMTU2LmV1LXdlc3QtMS5jb21wdXRlLmF=
tYXpvbmF3cy5jb206NTA1MC9hdXRoL3JlYWxtcy9jaGUiLCJhdWQiOiJodHRwOi8vZWMyLTM0LT=
I0MC0yMzYtMTU2LmV1LXdlc3QtMS5jb21wdXRlLmFtYXpvbmF3cy5jb206NTA1MC9hdXRoL3JlY=
Wxtcy9jaGUiLCJzdWIiOiJkOGNiMTYxMC05N2JlLTQ4NjAtOTRhOS1jYjY3NGQ3YzAzYjAiLCJ0=
eXAiOiJyZXNldC1jcmVkZW50aWFscyIsIm5vbmNlIjoiMzcxY2IyOTctMGI2My00Y2UwLTlmODk=
tMTE4YWVlMjc0OWI1IiwiYXNpZCI6IjU5MjgxNDliLTk0YzktNDMzOS1iZjNmLWFlYWFhMjlmYT=
MzZi5DM2I5Ti1pLVQzcy40MDE2MmM4Zi01YzQ0LTRiNjEtOTFjZi1hNmVhYzZiOWU2MWEiLCJhc=
2lkIjoiNTkyODE0OWItOTRjOS00MzM5LWJmM2YtYWVhYWEyOWZhMzNmLkMzYjlOLWktVDNzLjQw=
MTYyYzhmLTVjNDQtNGI2MS05MWNmLWE2ZWFjNmI5ZTYxYSJ9.2xM5bLAGD9wQOlug0n-2LNxbgY=
rbEGc5UlNMZTWixqljr3KvFGHSYl9FSl37gUPgSXd2xmIIpqXdBWMLt5tXag&amp;execution=
=3Df642498c-6067-4ad0-b4f3-387f13b9afaa&amp;client_id=3Dche-public&amp;tab_=
id=3DC3b9N-i-T3s&quot;&gt;Reset Password&lt;/a&gt;&lt;/span&gt;&lt;/p&gt;
        <p>Link will expire within 5 minutes.</p>
    </div>

<div style=3D"background-color: #292c2f; color: white; padding: 25px 10px;"=
>
    <a href=3D"http://www.eclipse.org/che/">
        <img src=3D"https://www.eclipse.org/che/images/logo-eclipseche.svg"=
 height=3D"32" alt=3D" " />
    </a>
</div>
</body>
</html>

------=_Part_2_922454530.1519904558025--
```

![image](https://user-images.githubusercontent.com/26924052/36843368-06ca5f56-1d4f-11e8-9092-d3891007e4e8.png)

### Reproduction Steps

1. Update SMTP settings in keycloak
2. Send "forgot password" email from che login

**OS and version:**    
Ubuntu 16.04
Che 6.1.0

**Diagnostics:**

**Comment by skabashnyuk** [comment]
@mmorhun  can you take a look?

**Comment by mmorhun** [comment]
@hkolvenbach thanks for reporting the problem.
Well, when I added that templates they worked fine, I checked them on gmail and another little known email service. So, I think it could be an issue with emails sender settings.
I do not mind adding `<#noautoesc>` or `no_esc` where needed (it might be more general solution), but we have to check it on both google mail and another one.

> If someone could point me to the location where this needs to be fixed, I could give it a try.

Here is the templates: https://github.com/eclipse/che/tree/master/dockerfiles/init/modules/keycloak/files/che/email
